homeassistant/components/deconz/sensor.py

```diff
@@ -31,6 +41,18 @@
 ATTR_DAYLIGHT = "daylight"
 ATTR_EVENT_ID = "event_id"
 
+DEVICE_CLASS = {
+    Alarm: "motion",
+    Consumption: "consumption",
+    Daylight: "daylight",
```

Comment:
Please remove the non existing device classes. Here are the available ones:
https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
Will do